ACadSharp/IO/DXF/DxfStreamWriter/DxfSectionWriterBase.cs

```diff
@@ -67,6 +57,24 @@ protected void writeCommonObjectData(CadObject cadObject)
 				//Add the dictionary in the object holder
 				this.Holder.Objects.Enqueue(cadObject.XDictionary);
 			}
+
+			if (cadObject.Reactors != null && cadObject.Reactors.Count > 0) {
```

Comment:
I do not consider Reactors to be fully implemented, if is needed for your implementation leave the writer as it is, but just for you to know that they may change the shape/behavior in the future.